torchci/pages/api/flaky-tests/disable.ts

```diff
@@ -0,0 +1,157 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import * as urllib from "urllib";
+
+import { getOctokit } from "lib/github";
+import fetchFlakyTests from "lib/fetchFlakyTests";
+import { FlakyTestData, IssueData } from "lib/types";
+import { supportedPlatforms } from "lib/bot/verifyDisableTestIssueBot";
+import fetchIssuesByLabel from "lib/fetchIssuesByLabel";
+
+
+const NUM_HOURS = 3;
+const owner: string = "pytorch";
+const repo: string = "pytorch";
+
+
+export default async function handler(
+    req: NextApiRequest,
+    res: NextApiResponse<void>
+) {
+    await disableFlakyTests();
+    res.status(200);
+}
+
+
+async function disableFlakyTests() {
+    const octokit = await getOctokit(owner, repo);
+    const flaky_tests = await fetchFlakyTests(`${NUM_HOURS}`);
+    const issues: IssueData[] = await fetchIssuesByLabel("skipped");
+    flaky_tests.forEach(async function(test: FlakyTestData) {
+        const issueTitle = getIssueTitle(test.name, test.suite);
+        const matchingIssues = issues.filter((issue) => issue.title === issueTitle);
+        if (matchingIssues.length !== 0) {
+            // There is a matching issue
+            const matchingIssue = matchingIssues[0];
+            if (matchingIssue.state === "open") {
+                const body = `Another case of flakiness has been found [here](${getLatestWorkflowURL(test.workflow_ids)}).
```

Comment:
This will get spammy if we are doing it every few hours and a test is consistently flaky. Maybe should just update the issue body with latest failures instead? Or maybe we should just direct people to a rich test-specific display on the HUD (as you suggested in the chat) and forgo doing update comments at all.

Reply:
Ah, that makes sense. Seems fine to leave then